### ISSUE TITLE
A little mixup in course about arithmetic and assignment operators

### ISSUE THREAD

**Issue by pauff** [jonasschmedtmann/complete-javascript-course#296]
@jonasschmedtmann 
I think there might be a little mixup in _JavaScript Fundamentals – Part 1 > Basic Operators_, where you classified `++`, `--` as assignment operators, but after checking [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#arithmetic_operators) I found out they should be **arithmetic** operators instead.

The code is in line 89 of the file below:
https://github.com/jonasschmedtmann/complete-javascript-course/blob/master/01-Fundamentals-Part-1/final/script.js

**Comment by akozdev** [comment]
Yeah, that's true. It will probably be fixed with some update in the future. Thanks!
